### PR TITLE
feat: Arbol case study

### PR DESCRIPTION
**Pull request by jessicaschilling**
**Still pending final internal and external stakeholder approval.**
Please don't merge (note: not marking as draft because we need build/deploy flow for staging!)

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@autonome -- For final review. Link:
https://bafybeigebdianpf3gf24nlx4juvgo5rp5fcnnxbtfsnbs5ygrvkj2pctce.ipfs.dweb.link/concepts/case-study-arbol

**Review by autonome — APPROVED** [review]
👍🏼 on final version, with one tweak requested for the intro sentence:

"agricultural entities and other weather-dependent parties with capital providers"

Would be better from an intro standpoint to provide examples of what these mean. Eg, farmers and investors.

Otherwise, ship it once it gets final approval from Arbol team.

**Comment by jessicaschilling** [comment]
Arbol approved with a few minor edits. Closing this issue.

**Pull request merged**
